explorer/app/components/StaticImage/StaticImage.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { StaticImage } from "./StaticImage";
+
+export default {
```

Comment:
Thanks so much for adding this!

I noticed an error when looking at the Logo stories in Storybook; would you mind taking a look?

<img width="994" alt="Screen Shot 2022-05-20 at 12 23 55 PM" src="https://user-images.githubusercontent.com/2446424/169597766-7df316f1-5e96-44d3-aab5-6c5086ba04f0.png">

Reply:
oops, my bad. Will fix!